pandas/io/excel.py

```diff
@@ -40,17 +40,16 @@
 _writers = {}
 
 _read_excel_doc = """
-Read an Excel table into a pandas DataFrame
+Read an Excel table into a pandas DataFrame.
 
```

Comment:
Can you add a extended summary giving a bit more information? Questions that came to my mind users could have and would be nice to answer there are:
-Which sheet/s in the Excel file will be loaded
- Which formats/versions of Excel do we support `xls`, `xlsx`?
- You can mention that the file to be opened can be in the local filesystem or a url.

Anything else you thing it can be helpful for users to know about this function before reading the parameters (where these questions will be answered in many cases)